src/org/omegat/gui/main/ProjectUICommands.java

```diff
@@ -1307,6 +1307,11 @@ public static void doWikiImport() {
             return;
         }
         try {
+            if (!HttpConnectionUtils.checkUrl(remoteUrl)) {
+                JOptionPane.showConfirmDialog(Core.getMainWindow().getApplicationFrame(),
+                        OStrings.getString("TF_WIKI_IMPORT_URL_ERROR"));
```

Comment:
```suggestion
                JOptionPane.showMessageDialog(Core.getMainWindow().getApplicationFrame(),
                        OStrings.getString("TF_WIKI_IMPORT_URL_ERROR"));
```

Reply:
@brandelune  Is it what you want to say?
https://docs.oracle.com/javase/tutorial/uiswing/components/dialog.html

Reply:
The question is: do we allow users to try an invalid URI or not?

If yes, the current dialog must be adapted and there should be a second dialog that says whether the resource existed or not.

If no, then the dialog you now suggest is what we need.

Reply:
When allowd invalid URL, you can enjoy exception stack trace.

Reply:
Ok.